ui/src/components/CardContainer/index.tsx

```diff
@@ -1,11 +1,28 @@
-import { Paper } from "@mui/material";
+import { Paper, PaperProps } from "@mui/material";
 import React from "react";
+import { classes } from "../../utils/styles";
 
 import styles from "./card-container.module.scss";
 
-export const CardContainer: React.FC = ({ children }) => {
+export const CardContainer: React.FC<PaperProps> = ({ children, ...rest }) => {
   return (
-    <Paper classes={{ root: styles.root }} elevation={1}>
+    <Paper
+      className={classes([styles.root, styles.bordered])}
+      elevation={1}
+      {...rest}
+    >
+      {children}
+    </Paper>
+  );
+};
+
+export const BorderlessCardContainer: React.FC = ({ children }) => {
+  return (
+    <Paper
+      classes={{ root: styles.root }}
+      elevation={1}
+      sx={{ borderWidth: 0 }}
```

Comment:
This background color should match the app body background.

```suggestion
      sx={{ borderWidth: 0, backgroundColor: "#fcfcfc" }}
```